--- a/src/src.pro
+++ b/src/src.pro
@@ -68,8 +68,8 @@ include(base/base.pri)
 !nowebui: include(webui/webui.pri)
 
 isEmpty(QMAKE_LRELEASE) {
-    win32: QMAKE_LRELEASE = $$[QT_INSTALL_BINS]/lrelease.exe
-    else: QMAKE_LRELEASE = $$[QT_INSTALL_BINS]/lrelease
+    win32: QMAKE_LRELEASE = $$[QT_HOST_BINS]/lrelease.exe
+    else: QMAKE_LRELEASE = $$[QT_HOST_BINS]/lrelease
     unix {
         equals(QT_MAJOR_VERSION, 5) {
             !exists($$QMAKE_LRELEASE): QMAKE_LRELEASE = lrelease-qt5
